webapp/components/admin_console/developer_settings.jsx

```diff
@@ -48,13 +48,13 @@ export default class DeveloperSettings extends AdminSettings {
                     label={
                         <FormattedMessage
                             id='admin.service.testingTitle'
-                            defaultMessage='Enable Testing: '
+                            defaultMessage='Enable Load Testing: '
```

Comment:
It's almost solely used to enable load testing, except it also hides the "disable all formatting" option I included last release as a safety valve for any formatting stuff that could crash the app. I think it should be left in the system console though since we use it for testing all the time so we need to enable it on the RC testing servers and CI servers